apps/site/assets/js/support-form.js

```diff
@@ -321,6 +322,12 @@ export function setupRequestResponse($) {
   });
 }
 
+export function setupADAResponse($) {
```

Comment:
question: What's the reason behind hiding the contact info section if the user selects "ADA complaint"? I'm not seeing anything about that in the Asana ticket, and there are no tests for this behavior.

Reply:
This should just be toggling the checkmark and not the contact info

Reply:
Is any JS at all required at this point?

Reply:
Yes, the setting of the checkbox determines wether the XML tag `<SWARELATED>` gets set as yes or no. I tested out removing the JS but in that case the form doesn't submit when the ADA complaint checkbox is not checked.

Reply:
Do you know why the form only submits with the JS? I could easily be missing something, but as a general case including a checkbox in a form seems like a pretty trivial thing that wouldn't require JS in order to function.

Reply:
Page has to be reloaded after each form submission. Not sure how often one would file multiple submissions in one sitting. Might create a new ticket if having to reload the page is deemed an issue.

Reply:
Huh. Is that an existing issue or something introduced by this change?

Reply:
Looks like an existing issue. Tried to back to back submissions on `dev-green` and the second doesn't go thru.

Reply:
Gotcha, thanks. In that case, I agree that creating a new, separate ticket seems appropriate.